src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1608,7 +1609,7 @@ bool InterpolateKey::operator==(const InterpolateKey &rhs) const {
         return false;
     if (nodeAttrs.outPrc != rhs.nodeAttrs.outPrc)
         return false;
-    if (nodeAttrs.layout != rhs.nodeAttrs.layout)
+    if (nodeAttrs.NCHWAsNHWC != rhs.nodeAttrs.NCHWAsNHWC)
```

Comment:
note: this is a deletion of repeat layout check, and add new attr, not a replacement.